mmocr/datasets/preparers/parsers/ic15_parser.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Tuple
+
+from ..data_preparer import DATA_PARSER
+from .base import BaseParser
+from .loaders import txt_loader
+
+
+@DATA_PARSER.register_module()
+class ICDAR2015TextDetParser(BaseParser):
```

Comment:
We do have XXXparsers in mmocr/utils/parsers.py. How about renaming it to `XXXAnnParser`